iamctl/pkg/utils/setup.go

```diff
@@ -0,0 +1,188 @@
+/**
+* Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+*
+* WSO2 LLC. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package utils
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type oAuthResponse struct {
+	AccessToken  string `json:"access_token"`
+	RefreshToken string `json:"refresh_token"`
+	Scope        string `json:"scope"`
+	TokenType    string `json:"token_type"`
+	Expires      int    `json:"expires_in"`
+}
+
+type ServerConfigs struct {
+	ServerUrl    string `json:"SERVER_URL"`
+	ClientId     string `json:"CLIENT_ID"`
+	ClientSecret string `json:"CLIENT_SECRET"`
+	TenantDomain string `json:"TENANT_DOMAIN"`
+	Username     string `json:"USERNAME"`
+	Password     string `json:"PASSWORD"`
+	Token        string `json:"TOKEN"`
+}
+
+type ToolConfigs struct {
+	KeywordMappings    map[string]interface{} `json:"KEYWORD_MAPPINGS"`
+	ApplicationConfigs map[string]interface{} `json:"APPLICATIONS"`
+}
+
+type Application struct {
+	Id          string `json:"id"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Self        string `json:"self"`
+}
+
+type AppList struct {
+	TotalResults int           `json:"totalResults"`
+	StartIndex   int           `json:"startIndex"`
+	Count        int           `json:"count"`
+	Applications []Application `json:"applications"`
+	Links        []string      `json:"links"`
+}
+
+var SERVER_CONFIGS ServerConfigs
+var TOOL_CONFIGS ToolConfigs
+
+func LoadConfigs(envConfigPath string) (baseDir string) {
+
+	baseDir = filepath.Dir(filepath.Dir(envConfigPath))
+	serverConfigFile := filepath.Join(envConfigPath, "serverConfig.json")
+	toolConfigFile := filepath.Join(envConfigPath, "toolConfig.json")
+
+	// Load configs from files.
+	SERVER_CONFIGS = loadServerConfigsFromFile(serverConfigFile)
+	TOOL_CONFIGS = loadToolConfigsFromFile(toolConfigFile)
+
+	return baseDir
+}
+
+func loadServerConfigsFromFile(configFilePath string) (serverConfigs ServerConfigs) {
+
+	configFile, err := os.Open(configFilePath)
+	if err != nil {
+		log.Fatalln(err.Error())
+	}
+	defer configFile.Close()
+
+	jsonParser := json.NewDecoder(configFile)
+	err = jsonParser.Decode(&serverConfigs)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	fmt.Println("Server configs loaded succesfully from the config file.")
```

Comment:
Any reason for using `fmt.Println` here? Better make all these info printed as info logs.

Reply:
Changed to log.Println